xbmc/games/dialogs/osd/DialogGameOSDHelp.cpp

```diff
@@ -17,6 +17,12 @@
 using namespace KODI;
 using namespace GAME;
 
+namespace
+{
+constexpr const char* HELP_COMBO =
```

Comment:
```suggestion
constexpr char HELP_COMBO[] =
```
I think this style has a slight advantage in that `std::begin/end` and `sizeof` can be used but I wouldn't enforce it.

Reply:
is const char[] ok?

Reply:
`constexpr` already implies `const` so the addition `const` isn't required.